packages/app/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "dependencies": {},
   "devDependencies": {
+    "@cypress-design/vue-icon": "^0.4.2",
```

Comment:
Unrelated but should this package be named `vue-icons`?

It looks like I'm importing a single icon, the Vue icon.

Reply:
There are actually 2 forms of components in `vue-icon` here, and I think you are right:
We should split `vue-icon` and `vue-icons` into two separate packages.

- `vue-icon` is the non-tree-shakable prop-defined icon component.
- `vue-icons` is the set of generated components that allow us to import icon per icon. 

Since the two packages were sharing so much code internally, I made them one and the same. We could see about aliasing it in the future.